src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import cn from 'classnames';
+import { Status } from '../../types/Status';
+
+type Props = {
+  selectedStatus: Status,
+  onHandleStatus:React.MouseEventHandler<HTMLAnchorElement>,
+  itemsLeftCount: number,
+  onDeleteCompletedTodo: () => Promise<void>,
+};
+
+export const Footer: React.FC<Props> = ({
+  selectedStatus,
+  onHandleStatus,
+  itemsLeftCount,
+  onDeleteCompletedTodo,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${itemsLeftCount} `}
+        items left
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter">
+        <a
+          href="#/"
+          className={
+            cn(
+              'filter__link',
+              { selected: selectedStatus === Status.default },
+            )
+          }
+          data-type={Status.default}
+          onClick={onHandleStatus}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={
+            cn(
+              'filter__link',
+              { selected: selectedStatus === Status.active },
+            )
+          }
+          onClick={onHandleStatus}
+          data-type={Status.active}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={
+            cn(
+              'filter__link',
+              { selected: selectedStatus === Status.completed },
+            )
+          }
+          onClick={onHandleStatus}
+          data-type={Status.completed}
+        >
+          Completed
+        </a>
+      </nav>
+
+      {/* don't show this button if there are no completed todos */}
```

Comment:
```suggestion
```
remove this comment but implement requirement in this comment

![Screenshot from 2023-06-11 15-17-47](https://github.com/mate-academy/react_todo-app-with-api/assets/33029864/565ae495-a483-4ad5-a4ae-d13d48feaa38)